### DISCUSSION TITLE
Error en mvn verifiy

### DISCUSSION THREAD

**Discussion by joacobruno** [dds-jv/foro#357] [Consultas de herramientas] [answered]
Hola estoy trabado con esto hace varios dias y nose como solucionarlo ya. Me podrian dar una mano? gracias

![image](https://user-images.githubusercontent.com/37660865/165646255-b8de62ed-ee23-4123-b6e5-3d893043d804.png)

**Comment by gastonprieto** [comment, marked as answer]
Buenas @joacobruno 

El error dice que no encuentra un compilador y te dice puede que tengas una JRE (runtime) en vez de la JDK (java developer kit), instalaste la jdk de Java 8? 

por favor cuando subas errores subi el texto, porquue es mas fácil de googlear o de que alguien lo busque a futuro. por ejemplo https://github.com/dds-jv/foro/discussions/356 pregunta lo mismo

**Reply by joacobruno** [reply]
Hola como va? Gracias por responder tan rapido. Lo acabo de solucionar. Igualmente gracias!

**Reply by gastonprieto** [reply]
nos contas como era @joacobruno asi si a otro le pasa, le queda la solución
